dotnet/AraGeometry/Geometry.cs

```diff
@@ -213,14 +213,14 @@ public static bool Coplanar(Vector3 v1, Vector3 v2, Vector3 v3, Vector3 v4, floa
         public static Vector3 Normal(this Face self)
             => Vector3.Normalize(Vector3.Cross(self.Binormal(), self.Tangent()));
         
-        public static IGeometry Mesh(int sidesPerFace, IArray<Vector3> vertices, IArray<int> indices = null)
-            => G3DExtensions.ToG3D(sidesPerFace, vertices, indices).ToIGeometry();
+        public static IGeometry Mesh(int sidesPerFace, IArray<Vector3> vertices, IArray<int> indices = null, IArray<Vector2> uvs = null)
```

Comment:
Should UVs be constrained to Vector2? Some packages like 3dsMax can consume a UVW Vector3 to modulate effects which read this secondary channel.

Reply:
You are right, they aren't really constrained to Vector2 overall in the Geometry API, but in this case it is a convenience builder function. 

3ds Max UVW are stored in Map channels which are special because they have an indirect mapping to corners, which is quite different than game engines, and many export formats, so in that case I label them literally as map channels.

Reply:
Ah, good to know. I was browsing a bit in there to see how I could declare some kind of map channel on a geometry. I'll do some more digging when I have the chance.